GettingStartedGuides/WPF_GettingStarted/MainWindow.xaml

```diff
@@ -0,0 +1,27 @@
+﻿<Window x:Class="WPFSample.MainWindow"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:local="clr-namespace:WPFSample"
+        xmlns:wv2="Windows.UI.Xaml.Controls"
+        mc:Ignorable="d"
+        Title="MainWindow" Height="450" Width="800">
+    
+    <DockPanel>
+
+        <DockPanel DockPanel.Dock="Top">
+            <Button x:Name="ButtonGo"
+            DockPanel.Dock="Right"
+            Click="ButtonGo_Click"
+            Content="Go"
+        />
+            <TextBox Name="addressBar"/>
+        </DockPanel>
+        
+        <wv2:WebView2 Name="webView"
+                  Source="https://jasonstephen15.github.io/"
```

Comment:
Would recommend using microsoft.com or bing as opposed to personal site.

Reply:
Good catch - def didn't mean that haha, must've been messing around with something - will change it!